iocell.8.txt

```diff
@@ -1196,6 +1196,17 @@ PROPERTIES
     Default: off
     Source: cpuset(1)
 
+  volumes=/host/dir[:/jail/dir[:mount_options]]
+
+  Mounts and unmounts directories from the host into the jail root using
+  nullfs(5).
+  'mount_options' are passed direcly to the mount command and defaults to "rw"
```

Comment:
I think it should default to 'ro' for security reasons, if not even fail, when no mode is provided.